sdk/core/loader/boot.S

```diff
@@ -126,7 +136,8 @@ start:
 	// All other registers will be cleared in the clear-regs block
 
 	// Pass the array of threadInfos as first argument.
-	csetbounds		ca0, csp, CONFIG_THREADS_NUM * BOOT_THREADINFO_SZ
+	addi			s1, s1, -16
```

Comment:
This one, too, could benefit from being an `EXPORT_ASSEMBLY_OFFSET`.

Reply:
These probably should be, I started that refactoring but it's probably better to finish in a separate NFC PR.